agent/pkg/dagent/utils/docker.go

```diff
@@ -35,8 +36,8 @@ import (
 
 const DockerClientTimeoutSeconds = 30
 
-var val time.Duration = (time.Duration(DockerClientTimeoutSeconds) * time.Second)
-var ContainerTimeout *time.Duration = &val
+var val time.Duration = (time.Duration(DockerClientTimeoutSeconds) * time.Second) //nolint:gochecknoglobals
```

Comment:
Will these stay? :cry:

Reply:
I can move them to `stopContainer` as locals.